yb-voyager/cmd/importSchemaYugabyteDB.go

```diff
@@ -37,6 +41,46 @@ func importSchemaInternal(exportDir string, importObjectList []string,
 		fmt.Printf("\nImporting %q\n\n", importObjectFilePath)
 		executeSqlFile(importObjectFilePath, importObjectType, skipFn)
 	}
+
+}
+
+/*
+Try re-executing each DDL from deffered list.
+If fails, silently avoid the error.
+Else remove from defferedSQLStmts list
+At the end, add the unsuccessful ones to a failedSqlStmts list and report to the user
+*/
+func importDefferedStatements() {
+	if len(defferedSqlStmts) == 0 {
+		return
+	}
+	log.Infof("Number of statements in defferedSQLStmts list: %d\n", len(defferedSqlStmts))
+
+	utils.PrintAndLog("\nExecuting the remaining SQL statements...\n\n")
+	maxIterations := len(defferedSqlStmts)
+	conn := newTargetConn()
+	defer conn.Close(context.Background())
+
+	var err error
+	// max loop iterations to remove all errors
+	for i := 1; i <= maxIterations && len(defferedSqlStmts) > 0; i++ {
+		for j := 0; j < len(defferedSqlStmts); j++ {
+			_, err = conn.Exec(context.Background(), defferedSqlStmts[j].formattedStmt)
+			if err == nil {
+				utils.PrintAndLog("%s\n\n", utils.GetSqlStmtToPrint(defferedSqlStmts[j].stmt))
+				// removing successfully executed SQL
+				defferedSqlStmts = append(defferedSqlStmts[:j], defferedSqlStmts[j+1:]...)
```

Comment:
This will skip running the next stmt. See https://go.dev/play/p/aEWHwckRXUS .

Reply:
done, now incrementing `j` in the `else` case